server/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.4"
 services:
   alfred:
-    image: mcr.microsoft.com/fluidframework/routerlicious/server:latest
+    image: ${REGISTRY_URL:-mcr.microsoft.com}/fluidframework/routerlicious/server:${ALFRED_IMAGE_TAG:-latest}
```

Comment:
Are we setting the default REGISTRY_URL or ALFRED_IMAGE_TAG anywhere in this PR? Or are they going to populated from elsewhere that already exists in the repo?

Reply:
No explicit defaults for the variables, but the way I'm using them applies the current defaults (`mcr.microsoft.com` for the registry, `latest` for the tags) if the variables are not provided. So this will be "if you don't set them, we'll keep using the same values as today; if you set them, we'll use that".